### DISCUSSION TITLE
[1.7.2] 2022-08-12

### DISCUSSION THREAD

**Discussion by darthcloud** [darthcloud/BlueRetro#353] [General]
### Bluetooth
* [HID] Assume report ID 1 if none provided by descriptor

### Wired
* [PCE] Fixup mouse support (Fixes #348)

### Global
* ESP-IDF updated to 5.0.0 2022-08-05

# Download are now on the new itch.io page: https://darthcloud.itch.io/blueretro

<hr /><em>This discussion was created from the release <a href='https://github.com/darthcloud/BlueRetro/releases/tag/v1.7.2'>[1.7.2] 2022-08-12</a>.</em>

**Comment by darthcloud** [comment]
I removed this version from the download list at itch.io, For now use v1.7.1.

This FW when installed on "fresh" ESP32 would reboot in loop due to a stack overflow. See #359 
Upgraded device were unaffected however.

Will release a v1.7.3 soon.

**Reply by fermel01** [reply]
Today I am going to perform an internal installation of ESP32 on my PS2 which has not been previously modified or installed on ESP32. What version of BlueRetro do you recommend then?
Thanks for your help!
